js/scripts.js

```diff
@@ -37,10 +37,20 @@ var app = {
       app.updateURL(url);
       $('input').val('');
 
+      app.setScroll(url);
+
     });
 
   },
 
+  setScroll: function(url) {
+    if(url === "welcome.html") {
+      $('html, body').animate( {scrollTop:0} );
```

Comment:
For consistency with the rest of the function calls in this code, please remove the spaces around the parens:

```js
$('html, body').animate({scrollTop:0});
```